src/scripts/modules/list.js

```diff
@@ -81,27 +81,26 @@ function renderDestination(destination) {
 
   //create figure caption
   const figcaption = document.createElement("figcaption");
-  figcaption.classList.add("caption");
+  figcaption.classList.add("caption__container");
   //set captions in a div
   const captionTxt = document.createElement("div");
-  captionTxt.classList.add("caption-text");
+  captionTxt.classList.add("caption__text");
   figcaption.appendChild(checkbox);
   figcaption.appendChild(captionTxt);
 
   captionTxt.insertAdjacentHTML(
     "beforeend",
-    `<h2 class='caption__city'>${destination.city}</h2>`
+    `<h2 class='img__caption'>${destination.city}</h2>`
   );
   captionTxt.insertAdjacentHTML(
     "beforeend",
-    `<h3 class='caption__country'>${destination.country}</h3>`
+    `<h3 class='img__date'>${destination.country}</h3>`
```

Comment:
Did you notice all the photos have "undefined" in their captions? Don't forge to fix that too

Reply:
Yeah it's because I haven't decided yet if I want to keep them so I didn't make those fields required on Parse, I'll see about it